kuma/api/v1/views.py

```diff
@@ -300,3 +301,22 @@ def bc_signal(request):
         serializer.save()
         return Response(serializer.validated_data, status=status.HTTP_201_CREATED)
     return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@never_cache
+@require_GET
+def get_user(request, username):
+    """
+    Returns a JSON response with a small subset of public information if a
+    user with the given username exists, otherwise returns a status code of
+    404. The case of the username is not important, since the collation of
+    the username column of the user table in MySQL is case-insensitive.
+    """
+    fields = ('username', 'title', 'fullname', 'timezone', 'locale')
+    try:
+        user = User.objects.only(*fields).get(username=username)
+    except User.DoesNotExist:
+        raise Http404('No user exists with the username "{}".'.format(username))
```

Comment:
```suggestion
        raise Http404(f'No user exists with the username "{username}".')
```

Reply:
Oh man, I forgot we're in the wonderful new world of Python 3 and its f-strings! 🎉